File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int i = 0 , j = nums.length-1 ;
        while(i<j){
            if(nums[j]==2) {
                j--;
                continue ;
            }
            if(nums[i]==2){
                int temp = nums[i];
                nums[i]=nums[j];
                nums[j]=temp;
                j--;
            }else i++;
        }
        i = 0 ;
        while(i<=j){
            if(nums[i]==1){
                int temp = nums[j];
                nums[j]=nums[i];
                nums[i]=temp;
                j--;
            }else i++;
        }
    }
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        z,o,t = 0 ,0 ,0
        n = len(nums)
        for i in range(0,n) :
            if nums[i] == 0 :
                nums[z],nums[i] = nums[i],nums[z]
                z+=1
        o = z
        for i in range (z,n):
            if nums[i] == 1 :
                nums[z],nums[i]=nums[i],nums[z]
                z+=1
        


